ci-operator/config/openshift/hive/openshift-hive-master.yaml

```diff
@@ -106,12 +119,31 @@ tests:
       - env: HIVE_IMAGE
         name: hive
       - env: RELEASE_IMAGE
-        name: release:latest
+        name: release:e2e-spoke
       from: src
       resources:
         requests:
           cpu: 100m
     workflow: generic-claim
+- as: e2e-4.9
+  steps:
+    cluster_profile: aws
+    dependencies:
+      RELEASE_IMAGE_LATEST: release:e2e-hub-4.9
+    test:
+    - as: test
+      cli: latest
```

Comment:
This (I think) seems to be [looking for](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/21007/rehearse-21007-pull-ci-openshift-hive-master-e2e-4.9/1425528272367652864#1:build-log.txt%3A48) an image called `stable-e2e-hub-4.9`. Why?

Reply:
The `cli` here is trying to import the `cli` image from the latest release. You likely want

```yaml
cli: e2e-spoke
```

Reply:
/me tries...